src/Libraries/CoreNodesUI/Watch.cs

```diff
@@ -1,228 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-
-using Dynamo.Controls;
-using Dynamo.Core.Threading;
```

Comment:
For reasons I don't understand, this did not get a proper diff, nor did WatchNodeViewCustomization.  It is so significantly modified that it probably wouldn't help...